src/pages/workspace/workflows/approvals/ApprovalWorkflowEditor.tsx

```diff
@@ -0,0 +1,104 @@
+import React, {useCallback} from 'react';
+import {View} from 'react-native';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import ScrollView from '@components/ScrollView';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+import type {ApprovalWorkflowOnyx} from '@src/types/onyx';
+
+type ApprovalWorkflowEditorProps = {
+    approvalWorkflow: ApprovalWorkflowOnyx;
+    policyID: string;
+};
+
+function ApprovalWorkflowEditor({approvalWorkflow, policyID}: ApprovalWorkflowEditorProps) {
+    const styles = useThemeStyles();
+    const {translate, toLocaleOrdinal} = useLocalize();
+
+    const approverDescription = useCallback(
+        (index: number) =>
+            approvalWorkflow.approvers.length > 1 ? `${toLocaleOrdinal(index + 1, true)} ${translate('workflowsPage.approver').toLowerCase()}` : `${translate('workflowsPage.approver')}`,
```

Comment:
Nit: I guess we can't extract this code to WorkflowUtils, because we use both `useCallback` and `useLocalize` hooks? The same exact function is created in `ApprovalWorkflowSection`

Reply:
Yep, it could be cumbersome. I'll leave it as is 👍